### ISSUE TITLE
UNKNOWN ENTITY ERROR if ID starts with a number

### ISSUE THREAD

**Issue by mkeays** [ontodev/robot#789]
Hello, I'm getting this error and I'm not sure why:

```
echo "ODK Makefile version: v1.2.23 (this is the version of the ODK with which this Makefile was generated, not the version of the ODK you are running)" &&\
echo "ROBOT version (ODK): " && robot --catalog catalog-v001.xml --version
ODK Makefile version: v1.2.23 (this is the version of the ODK with which this Makefile was generated, not the version of the ODK you are running)
ROBOT version (ODK):
ROBOT version 1.7.2
robot --catalog catalog-v001.xml template -i ../templates/support.owl \
--add-prefix "hgnc: http://identifiers.org/hgnc:" \
--add-prefix "antibodyregistry: http://identifiers.org/antibodyregistry:" \
--add-prefix "complexportal: http://identifiers.org/complexportal:" \
--template ../templates/markers.tsv convert -f ofn -o ../templates/markers.owl
UNKNOWN ENTITY ERROR could not interpret 'hgnc:12523' in row 3, column 2 ("ID") in table "../templates/markers.tsv".
For details see: http://robot.obolibrary.org/template#unknown-entity-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
camporto.Makefile:7: recipe for target '../templates/markers.owl' failed
make: *** [../templates/markers.owl] Error 1
```

The above mentioned `../templates/markers.tsv` looks like this:

```
Genus | ID | label
-- | -- | --
C % | ID | A rdfs:label
SO:0000704 | hgnc:12523 | SCGB1A1
SO:0000704 | hgnc:2770 | desmin
SO:0000704 | hgnc:10806 | sarcoglycan beta
```

I am running this command to set up the `hgnc` prefix:

```
../templates/%.owl: ../templates/%.tsv
	$(ROBOT) template -i ../templates/support.owl \
	--add-prefix "hgnc: http://identifiers.org/hgnc:" \
	--add-prefix "antibodyregistry: http://identifiers.org/antibodyregistry:" \
	--add-prefix "complexportal: http://identifiers.org/complexportal:" \
	--template $< convert -f ofn -o $@
```

I noticed that if I add an "a" to the start of the numeric ID, e.g. `hgnc:a12523` etc, then this error goes away, but it means the IDs are incorrect obviously. Am I doing something wrong here?
Thanks!

cc @dosumis

**Comment by dosumis** [comment]
It looks like this might come from requiring a valid [Qname](https://en.wikipedia.org/wiki/QName)*. Start character can't be numeric.  This means we can't use many of the IRI recipes from identifiers.org - which provides a very convenient way to map all sorts of IDs that don't have standard IRIs.  

From the Qname spec, it looks like ':' is a valid first character.  

@mkeays - can you try this in the template: `hgnc::12523`

+ this in the  ROBOT template command --add-prefix "hgnc: http://identifiers.org/hgnc" ?

\* thanks to Maria for pointing me to this: https://github.com/ontodev/robot/pull/783

**Comment by mkeays** [comment]
Thanks @dosumis -- I tried that but this time I got:

```
echo "ODK Makefile version: v1.2.23 (this is the version of the ODK with which this Makefile was generated, not the version of the ODK you are running)" &&\
echo "ROBOT version (ODK): " && robot --catalog catalog-v001.xml --version
ODK Makefile version: v1.2.23 (this is the version of the ODK with which this Makefile was generated, not the version of the ODK you are running)
ROBOT version (ODK):
ROBOT version 1.7.2
robot --catalog catalog-v001.xml template -i ../templates/support.owl \
--add-prefix "hgnc: http://identifiers.org/hgnc" \
--add-prefix "antibodyregistry: http://identifiers.org/antibodyregistry:" \
--add-prefix "complexportal: http://identifiers.org/complexportal:" \
--template ../templates/camporto_lung.tsv convert -f ofn -o ../templates/camporto_lung.owl
MANCHESTER PARSE ERROR the expression 'expresses some hgnc::7515' at row 3, column 6 in table "../templates/camporto_lung.tsv" cannot be parsed: encountered unknown 'hgnc::7515'
For details see: http://robot.obolibrary.org/template#manchester-parse-error
Use the -vvv option to show the stack trace.
Use the --help option to see usage information.
camporto.Makefile:7: recipe for target '../templates/camporto_lung.owl' failed
make: *** [../templates/camporto_lung.owl] Error 1
```

**Comment by jamesaoverton** [comment]
I think the first issue you described is the same as #782, and I believe we have a fix in #783. Can you please test with the `robot.jar` from here <https://build.obolibrary.io/job/ontodev/job/robot/job/782-fix/> and let us know if that helps?

I don't think the double colon trick will work.

**Comment by mkeays** [comment]
Thanks @jamesaoverton -- I tried the `robot.jar` you suggested and it worked as expected!

**Comment by jamesaoverton** [comment]
Ok, great. I merged #783, and we'll make a new ROBOT release soon. So I'll close this issue.
